src/descriptors/analysis.rs

```diff
@@ -117,10 +119,12 @@ impl DescKeyChecker {
             if let Some(ref origin) = xpub.origin {
                 let der_paths = xpub.derivation_paths.paths();
                 let first_der_path = der_paths.first().expect("Cannot be empty");
+                let second_der_path = &der_paths[1];
```

Comment:
Why would this vector always have at least two elements?

Reply:
I guessed as `xpub` is a `MultiXPub` it should have at least 2 paths, but yeah it seems we can create a single element `DerivPaths`.